packages/shared_preferences/shared_preferences/example/integration_test/shared_preferences_test.dart

```diff
@@ -97,5 +89,33 @@ void main() {
       // The last write should win.
       expect(preferences.getInt('int'), writeCount);
     });
+  }
+
+  group('$SharedPreferences', () {
```

Comment:
Remove the `$`. I've been removing that pattern everywhere incrementally, because it confuses the VS Code UI.